src/uct/ib/rc/base/rc_ep.c

```diff
@@ -266,7 +281,7 @@ ucs_arbiter_cb_result_t uct_rc_ep_process_pending(ucs_arbiter_t *arbiter,
         return UCS_ARBITER_CB_RESULT_NEXT_GROUP;
     } else {
         ep = ucs_container_of(ucs_arbiter_elem_group(elem), uct_rc_ep_t, arb_group);
-        if (ep->available <= 0) {
+        if (! uct_rc_ep_has_tx_resources(ep)) {
```

Comment:
so it means that if RDMA_WRITE operation is on the pending queue, it would have to wait for credits, but when it is called in the first time, it can be sent w/o fc credits..